calculatorUICOOP/ViewModels/MainPageViewModel.cs

```diff
@@ -148,7 +148,10 @@ public void DeleteLastAppend()
         {
             if (Input.Length > 1)
             {
-                Input = Input.Remove(Input.Length - 1);
+                if (Input.Length == 2 && Input.Contains("-"))
+                    ClearEntry();
```

Comment:
IMHO the `DeleteLastAppend` shouldn't not depend on `ClearEntry`, as changes in `ClearEntry` may also make unexpected changes to the behavior of `DeleteLastAppend`.
What do you think, @nickmartin1ee7?

Reply:
Thank you so much for contributing, @JBSE 🥳

Reply:
Good point!

I wasn't even thinking about that!